Doc/library/stdtypes.rst

```diff
@@ -1129,7 +1129,7 @@ Lists
 
 Lists are mutable sequences, typically used to store collections of
 homogeneous items (where the precise degree of similarity will vary by
-application).
+application).  List elements are accessed with a O(1).
```

Comment:
Suggestion: "List elements are accessed with O(1) time complexity" or "Accessing list elements has O(1) time complexity".

Reply:
IMHO, mentioning "time complexity" is definitely an improvement, I'll rephrase, thanks :)